openedx/core/djangoapps/user_api/accounts/image_helpers.py

```diff
@@ -37,21 +38,20 @@ def _make_profile_image_name(username):
     return hashlib.md5(settings.PROFILE_IMAGE_SECRET_KEY + username).hexdigest()
 
 
-def _get_profile_image_filename(name, size):
+def _get_profile_image_filename(name, size, file_extension='jpg'):
     """
     Returns the full filename for a profile image, given the name and size.
     """
-    return '{name}_{size}.jpg'.format(name=name, size=size)
+    return '{name}_{size}.{file_extension}'.format(name=name, size=size, file_extension=file_extension)
 
 
-def _get_profile_image_urls(name):
+def _get_profile_image_urls(name, storage, file_extension='jpg'):
```

Comment:
We decided on using jpeg as our standard file format for profile images - when we crop & thumbnail user-uploaded images we output jpegs. I think this should be constant-ized so that it seems less like a dynamically configurable thing. Another option would be to convert these default images to jpeg, and just always expect jpegs.